src/vs/editor/contrib/suggest/suggestWidget.ts

```diff
@@ -427,7 +429,7 @@ export class SuggestWidget implements IContentWidget, IListVirtualDelegate<Compl
 
 	// Editor.IContentWidget.allowEditorOverflow
 	readonly allowEditorOverflow = true;
-	readonly suppressMouseDown = true;
+	readonly suppressMouseDown = false;
```

Comment:
This makes sure `details` widget get onclick events (otherwise `textInputArea` gets it).